homeassistant/components/zha/core/device.py

```diff
@@ -321,15 +321,18 @@ def async_get_clusters(self):
         }
 
     @callback
-    def async_get_zha_clusters(self):
-        """Get zigbee home automation clusters for this device."""
-        from zigpy.profiles.zha import PROFILE_ID
+    def async_get_std_clusters(self):
+        """Get ZHA and ZLL clusters for this device."""
+        from zigpy.profiles import zha, zll
         return {
             ep_id: {
                 IN: endpoint.in_clusters,
                 OUT: endpoint.out_clusters
             } for (ep_id, endpoint) in self._zigpy_device.endpoints.items()
-            if ep_id != 0 and endpoint.profile_id == PROFILE_ID
+            if ep_id != 0 and (
+                endpoint.profile_id == zha.PROFILE_ID or
```

Comment:
`endpoind.profile_id in (zha.PROFILE_ID, zll.PROFILE_ID)` ?

Reply:
sure, why not... 1 min

Reply:
done